projects/demo/src/app/alert/angular/alert-angular-app-level-alerts.demo.html

```diff
@@ -51,6 +51,15 @@
         <span class="title">Header</span>
       </div>
     </header>
+
+    <clr-alerts>
```

Comment:
This is great for having in the demo application. Adding the specific CDE in a comment could help us understand why this particular case is necessary. 

I'd also like a test to assert that an error is not logged to prevent future regressions.

Reply:
Updated test in commit 9b9b33f